test/app/test_GUI.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+import pygame
+
+from src.app.GUI import GUI
+from unittest.mock import MagicMock, patch
+
+
+class Test_GUI(unittest.TestCase):
+    def setUp(self) -> None:
+        self.width = 400
+        self.height = 400
+        self.GUI = GUI(self.width, self.height)
+        self.zemlja = MagicMock()
+        self.snake = MagicMock()
+        self.zemlja.snake = self.snake
+
+    @patch('pygame.event.get', return_value=[
```

Comment:
Tukaj z patch-om simuliraš uporabniški input in effekt ki ga uporabniški input naredi je da spremeni elemente domenskega prostora in to bi moral tudi tukaj testirati.